app/styles/components/bc-tabs.scss

```diff
@@ -28,7 +28,7 @@
 			@include bc-transition(all .5s ease-in-out);
 			@include bc-box-sizing;
 			position: relative;
-			padding: 15px 16px;
+			padding: 18px 18px;
```

Comment:
The padding is meant to handle the actual tabs spacing from each other. To adjust the offset from the top of the page it needs to be handled in the tabs-header class.